src/main/sensors/battery.c

```diff
@@ -35,107 +35,102 @@
 
 #include "fc/config.h"
 #include "fc/rc_controls.h"
-#include "fc/runtime_config.h"
 
 #include "io/beeper.h"
 
 #include "sensors/battery.h"
-#include "sensors/esc_sensor.h"
 
+/**
+ * terminology: meter vs sensors
+ *
+ * sensors are used to collect data.
+ * - e.g. voltage at an MCU ADC input pin, value from an ESC sensor.
+ *   sensors require very specific configuration, such as resistor values.
+ * meters are used to process and expose data collected from sensors to the rest of the system.
+ * - e.g. a meter exposes normalized, and often filtered, values from a sensor.
+ *   meters require different or little configuration.
+ *   meters also have different precision concerns, and may use different units to the sensors.
+ *
+ */
 
```

Comment:
What I said was:
Although this distinction is logical, it doesn't fully align with other parts of the code. So, for example, the gyro in the sensors directory returns calibrated, scaled and filtered data.

I don't suggest you change the code, but perhaps you could update that description to reflect this.